<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasureDialog</class>
 <widget class="QDialog" name="MeasureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected measurement: 0 - 100 km/h</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelTimeHeader">
        <property name="text">
         <string>Time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSpeedHeader">
        <property name="text">
         <string>Speed:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelTime">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSpeed">
        <property name="text">
         <string>86 km/h</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonAbort">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
